aedttest/logger.py

```diff
@@ -42,8 +42,8 @@ def set_logger(logging_file, level=logging.DEBUG, pyaedt_module=None):
 
     if pyaedt_module is not None:
         if level > logging.DEBUG:
-            pyaedt_module.aedt_logger.ENABLE_LOGGER = False
+            pyaedt_module.settings.enable_screen_logs = False
```

Comment:
what does screen logs mean?

does it mean all logging or only logging to stdout ?

Reply:
Only logging to stdout. The AEDT logging is disabled with settings.enable_desktop_logs

Reply:
Done. The command added disables both.